pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -378,69 +378,43 @@ private boolean isDistinct(final AggregationFunction[] aggregationFunctions) {
    * @param dataSchema data schema.
    */
   @SuppressWarnings("unchecked")
-  private void setAggregationResults(BrokerResponseNative brokerResponseNative,
+  private void setAggregationResults(BrokerRequest brokerRequest, BrokerResponseNative brokerResponseNative,
```

Comment:
bad github review -- I have only moved the if condition to beginning of the function and rest everything is unchanged. It shows as if the function has been rewritten